pkg/helpers/apply/apply.go

```diff
@@ -45,6 +45,7 @@ var (
 func ApplyDeployments(
 	kubeClient kubernetes.Interface,
 	reader asset.ScenarioReader,
+	customFuncMap template.FuncMap,
```

Comment:
I am not sure the purpose of this. Is it to render the yaml template in a customized way?